PowerShell/ScubaGear/Modules/CreateReport/CreateReport.psm1

```diff
@@ -210,8 +210,22 @@ function New-Report {
 
     # Handle AAD-specific reporting
     if ($BaselineName -eq "aad") {
+        
+        # Load the CSV file
+        $csvData = Import-Csv -Path "Product Names for Licensing.csv"
```

Comment:
Can we update the unit test for this case (e.g. by mocking out Import-Csv to avoid loading the file)?  I think we added a decent test for this table IIRC.

Reply:
Resolved with Grant via slack